less/v2/character.less

```diff
@@ -694,6 +694,10 @@
               }
             }
           }
+
+          .death-tab > i {
+            font-size: 12px;
+          }
```

Comment:
Should this be `.card-tab > i`? We have some general `.card-tab` styles around line  367.